lib/state/auth/constants/constants.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/foundation.dart' show immutable;
+
+@immutable
+class Constants {
```

Comment:
A suggestion would be to use this class more with the other existing hardcoded values so it'a more maintainable